alert-common/src/main/java/com/synopsys/integration/alert/common/channel/message/ChannelMessageParser.java

```diff
@@ -77,14 +79,24 @@ public String createHeader(MessageContentGroup messageContentGroup) {
         return headerSeparator;
     }
 
-    public String getCommonTopic(MessageContentGroup messageContentGroup) {
-        return createLinkableItemString(messageContentGroup.getCommonTopic(), true) + getLineSeparator();
+    public String getCommonTopic(MessageContentGroup messageContent, ItemOperation nullableTopLevelAction) {
```

Comment:
Why not add @Nullable, javax.annotation.Nullable, to the nullableTopLevelAction?

``` java
public String getCommonTopic(MessageContentGroup messageContent, @Nullable ItemOperation nullableTopLevelAction) {
```

Reply:
We don't have a standardized use of this and even if it is null, it is used in a null-safe way.